front/app/containers/Admin/projects/project/analysis/Insights/messages.ts

```diff
@@ -106,4 +106,9 @@ export default defineMessages({
     id: 'app.containers.Admin.projects.project.analysis.aiSummary',
     defaultMessage: 'AI summary',
   },
+  aiSummaryTooltip: {
+    id: 'app.containers.Admin.projects.project.analysis.aiSummaryTooltip',
+    defaultMessage:
+      'This is AI-generated content. It is based on the data you have selected and may not be 100% accurate. Please review and cross-reference with the actual inputs for accuracy. Be aware that the accuracy is likely to improve if the number of selected inputs is reduced.',
```

Comment:
I think it's good, only point of doubt is

> It is based on the data you have selected

For many people, when they'll first see this they won't have selected any data yet, might be confusing? I would just drop this part of the sentence.

We also refer to selected inputs  at the end of the text, but I think there it's fine.